lib/main.js

```diff
@@ -223,6 +222,15 @@ const Hydrogen = {
     autorun(() => {
       this.emitter.emit("did-change-kernel", store.kernel);
     });
+
+    when(
+      "When the first kernel started open output area",
+      () => store.runningKernels.length > 0,
+      () => {
+        if (!atom.config.get("Hydrogen.outputAreaDefault")) return;
+        openOrShowDock(OUTPUT_AREA_URI);
+      }
+    );
```

Comment:
The string argument here is a "debug name" that will appear in the log when the reaction fires (only in dev mode when `mobx-react-devtools` enabled.